modular_skyrat/modules/antilag/code/admin_procs.dm

```diff
@@ -0,0 +1,57 @@
+/client/proc/kick_afk()
+	set category = "Admin"
+	set name = "Kick ALL AFK Clients"
+	set desc = "Kicks all clients who are AFK, both ingame and in the lobby."
+
+	var/kick_type_choice = tgui_alert("Do you want to kick all AFK clients, or only those observing or in the lobby?", "All of them", "Only those in the lobby/observing", "Cancel")
+
+	if(!kick_type_choice || kick_type_choice == "Cancel")
+		return
+
+	var/list/kicked_client_names = list()
+
+	if(kick_type_choice == "All of them")
+		for(var/client/iterating_client as anything in GLOB.clients)
+			if(iterating_client.is_afk())
```

Comment:
```suggestion
		for(var/client/iterating_client as anything in GLOB.clients)
			if(iterating_client?.is_afk())
```

Reply:
or maybe just a if(!iterating_client)

Reply:
that global list should never have a null in it

Reply:
Alright, just making sure.

Reply:
Probably good to have a check tho